internal/pkg/cli/job_init.go

```diff
@@ -214,6 +228,35 @@ func (o *initJobOpts) Ask() error {
 	return nil
 }
 
+// hasOnlyPrivateSubnets returns the list of environments names deployed that contains only private subnets.
+func (o *initJobOpts) hasOnlyPrivateSubnets() ([]string, error) {
```

Comment:
It seems like this method is very similar to the one in `svc_init`, can we try to reduce the duplication?